release-notes/opensearch-neural-search.release-notes-2.19.0.0.md

```diff
@@ -0,0 +1,30 @@
+## Version 2.19.0.0 Release Notes
+
+Compatible with OpenSearch 2.19.0
+
+### Features
+* Pagination in Hybrid query ([#1048](https://github.com/opensearch-project/neural-search/pull/1048))
+* Implement Reciprocal Rank Fusion score normalization/combination technique in hybrid query ([#874](https://github.com/opensearch-project/neural-search/pull/874))
+
+### Bug Fixes
+* Address inconsistent scoring in hybrid query results ([#998](https://github.com/opensearch-project/neural-search/pull/998))
+* Fix bug where ingested document has list of nested objects ([#1040](https://github.com/opensearch-project/neural-search/pull/1040))
+* Fixed document source and score field mismatch in sorted hybrid queries ([#1043](https://github.com/opensearch-project/neural-search/pull/1043))
+* Update NeuralQueryBuilder doEquals() and doHashCode() to cater the missing parameters information ([#1045](https://github.com/opensearch-project/neural-search/pull/1045)).
+* Fix bug where embedding is missing when ingested document has "." in field name, and mismatches fieldMap config ([#1062](https://github.com/opensearch-project/neural-search/pull/1062))
+* Fix explain exception in hybrid queries with partial subquery matches ([#1123](https://github.com/opensearch-project/neural-search/pull/1123))
+* Handle pagination_depth when from =0 and removes default value of pagination_depth ([#1132](https://github.com/opensearch-project/neural-search/pull/1132))
+* Fix single shard pagination issue of from ([#1140](https://github.com/opensearch-project/neural-search/pull/1140))
```

Comment:
do we need these two for pagination as part of the release notes? same reasoning, we're releasing pagination in 2.19, can be counted as part of the implementation as those are internal findings. @vibrantvarun WDYT?

Reply:
No we should remove 
```
* Fix single shard pagination issue of from ([#1140](https://github.com/opensearch-project/neural-search/pull/1140))

```

Reply:
Sure. Will remove line 16, 17